Backend/src/main/java/com/br/sga/domain/Evento.java

```diff
@@ -1,15 +1,7 @@
 package com.br.sga.domain;
 
 
-import jakarta.persistence.Column;
-import jakarta.persistence.Entity;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.GenerationType;
-import jakarta.persistence.Id;
-import jakarta.persistence.JoinColumn;
-import jakarta.persistence.ManyToOne;
-import jakarta.persistence.SequenceGenerator;
-import jakarta.persistence.Table;
+import jakarta.persistence.*;
```

Comment:
Não adicionar imports assim. Importar somente o que for necessário. Provável que seja uma configuração do Intellij.. deixar como estava antes